### DISCUSSION TITLE
What is the code needed for the Exit button on the page?

### DISCUSSION THREAD

**Discussion by tafemq** [tafecd/Tafe230120#5] [General]
I had a hard time finding how to exit the page on my currency conversion calculator. Google has not been helpful much.
I first used the Application quit, but it closed the entire application, not just my page.
I finally found the right way, which closed my page and returned to the initiating page (main menu) via:
Frame.goBack();

What was your way of exiting your mortgage calculator?

**Comment by tafecd** [comment]
I used slightly different code: `this.Frame.Navigate(typeof(MainMenu));`
I think we'll be fine as long as both work. :)
